src/components/Card/Card.test.jsx

```diff
@@ -0,0 +1,39 @@
+import React from 'react';
+import { render, cleanup } from '@testing-library/react';
+import { BrowserRouter } from 'react-router-dom';
+
+import Card from '.';
+
+beforeEach(cleanup);
+
+const video = {
+  snippet: {
+    title: 'wizeline',
+    description: 'video wizeline',
+    thumbnails: {
+      medium: {
+        url: 'http://www.youtube.com/watch?v',
+      },
+    },
+  },
+  id: {
+    videoId: '01',
+  },
+};
+
+describe('VideoGrid', () => {
+  it('Should render Text wizeline', () => {
+    const { getByText } = render(
```

Comment:
instead of destructuring the return of the `render` method, it is better to use `screen.getByText` 

source: [Common Mistakes with React Testing Library](https://kentcdodds.com/blog/common-mistakes-with-react-testing-library#not-using-screen)